### PR TITLE
Feature/map 13

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -251,7 +106,7 @@
           "input": {
             "defaultBasemapStyle": {
               "name": "Positron",
-              "styleFile": "${ARLAS_MAP_STYLE}"
+              "styleFile": "http://demo.arlas.io:82/styles/positron/style.json"
```
Comment (sebbousquet) [inline src/config.json:109]: I know it's for dev but  maybe you should add a comment to avoid forget it

File: src/config.map.json [hunk 2]
```diff
@@ -39,33 +54,15 @@
...
-        "aggregation"
-      ]
+      }
     },
     {
-      "id": "cluster-heat",
+      "id": "cluster_heat",
       "type": "heatmap",
-      "source": "data_source",
+      "source": "cluster-cluster_heatmap",
```
Comment (sebbousquet) [inline src/config.map.json:62]: It's look like a bit redundant `cluster-cluster...`

Comment (MohamedHamouGisaia) [reply src/config.map.json:62]: I will treat it in the next PR

File: src/config.json (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/config.json:109]: I will treat it in the next PR

**Pull request merged**